dxr/static/templates/path_line.html

```diff
@@ -1,10 +1,13 @@
 {%- if not is_first_or_only -%}
-<span class="path-separator">/</span>
+    <span class="path-separator">/</span>
 {%- endif -%}
 {%- if not is_binary or is_dir -%}
-<a href="{{ url }}" {%- if is_dir %} data-path="{{ data_path }}" {% endif %}>
+    <a href="{{ url }}" {%- if is_dir %} data-path="{{ data_path }}" {% endif %}>
 {%- endif -%}
-    {{ display_path }}
+{# Remark: it is possible for <b> tags to be unbalanced within fragments if it opens in one
+   fragment and closes in another, but we avoid any issue by showing the entire path as a
+   single element. #}
+{{ display_path|safe }}
```

Comment:
Also we trust our escaping in [result_lines.html](http://git.io/vmZnB), which follows almost the exact same execution flow as paths (only without the splitting on slash) since they share the query.highlight method.